.github/workflows/coverage.yml

```
@@ -33,7 +33,7 @@ jobs:
         id: coverage
 
       - name: Report code coverage
-        uses: zgosalvez/github-actions-report-lcov@v3
+        uses: zgosalvez/github-actions-report-lcov@v4.1.22
```

It's better to use major versions so we don't have to update for each minor versions.
```suggestion
        uses: zgosalvez/github-actions-report-lcov@v4
```